packages/tools/src/commands/integrity/readme.ts

```diff
@@ -51,19 +64,25 @@ const getReadmeToc = (readmeContent: string) => {
     return toc;
 };
 
-const getPluginMetadata = async (plugin: Workspace) => {
+const getPluginMetadata = async (plugin: Workspace): Promise<PluginMetadata> => {
     const { CONFIG_KEY } = await require(path.resolve(ROOT, plugin.location, 'src/constants.ts'));
     // Load plugin's README.md file.
     const readmePath = path.resolve(ROOT, plugin.location, 'README.md');
     const readme = fs.readFileSync(readmePath, 'utf-8');
     // Get the title and the first paragraph.
     const title = readme.match(/# (.*)/)?.[1].replace(` ${MD_TOC_OMIT_KEY}`, '') || '';
     const intro = readme.match(/# .*\n\n(.*)/)?.[1] || '';
-    return { title, intro, key: CONFIG_KEY };
+    const config = readme.match(/## Configuration[\s\S]*?```[^\n\r]+\n([\s\S]*?)\n```/)?.[1] || '';
```

Comment:
got a bit puzzled by this `[\s\S]*?` but If understood correctly the diff with `.*` is that it matches the line breaks right?

Reply:
Yes, exactly, I'll add a comment to clarify this.